draft-ietf-oauth-status-list.md

```diff
@@ -973,6 +975,14 @@ By default, this specification only supports providing Status List information f
 
 There are strong privacy concerns that have to be carefully taken into consideration when providing a mechanism that allows historic requests for status information - see [](#privacy-relying-party) for more details. Support for this functionality is optional and Implementers are RECOMMENDED to not support historic requests unless there are strong reasons to do so and after carefully considering the privacy implications.
 
+## Status Types {#privacy-status-types}
+
+As previously explained, there is the potential risk of observability by Relying Parties and Outsiders. That means that any Status Type that transports special information about a Token can leak information to other parties. This documents defines one additional Status Type with "SUSPENDED" that conveys such additional information. Depending on the use-case, suspended could for example provide information that an authorization in the Token is suspended, but the token itself is still valid.
```

Comment:
Maybe we should be specific around where this is "previous explained" e.g by linking to a section.